src/components/Toolbar.js

```diff
@@ -190,11 +208,13 @@ class Toolbar extends React.Component<Props, State> {
   }
 
 
+  /** @returns An array of top position offsets that the toolbar is allowed to stop on. */
+
   getStops(): number[] {
-    const minimalTopPosition = this.getMinimalTopPosition();
+    const maximalTopPosition = this.getMaximalTopPosition();
     const stops = uniq([
-      minimalTopPosition,
-      Math.max(minimalTopPosition, Math.floor(this.state.viewportSize.height / 2)),
+      maximalTopPosition,
+      Math.max(maximalTopPosition, Math.floor(this.state.viewportSize.height / 2)),
```

Comment:
Should this not be Math.min? Now it returns always something larger or equal to maximalTopPosition